plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -447,16 +448,17 @@ export const SettingsPaymentsMain = () => {
 	};
 
 	const morePaymentOptionsLink = (
-		<Button
-			variant={ 'link' }
-			target="_blank"
-			href="https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/"
-			className="more-payment-options-link"
-			onClick={ trackMorePaymentsOptionsClicked }
-		>
-			<img src={ assetUrl + '/icons/external-link.svg' } alt="" />
-			{ __( 'More payment options', 'woocommerce' ) }
-		</Button>
+		<TrackedLink
+			message={ __(
+				// translators: {{Link}} is a placeholder for a html element.
+				'Visit {{Link}}The WooCommerce Marketplace{{/Link}} to find additional payment providers.',
+				'woocommerce'
+			) }
+			onClickCallback={ trackMorePaymentsOptionsClicked }
+			targetUrl={ getAdminLink(
```

Comment:
The Marketplace is a behind a (hidden) feature flag. While the UI may not be available, all it takes is this snippet to disable it:
```
add_filter( 'pre_option_woocommerce_feature_marketplace_enabled',function ( $enabled ) { return 'no'; } );
```
If the feature is disabled, we should fallback on sending folks to woocommerce.com. Something like this would do it:

```
targetUrl={
				isFeatureEnabled( 'marketplace' )
					? getAdminLink(
							'admin.php?page=wc-admin&tab=extensions&path=/extensions&category=payment-gateways'
					  )
					: 'https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/'
			}
```

Reply:
Thanks, @vladolaru. I wasn't aware the marketplace could be disabled so easily.

In that case, we'd need to add this fallback to the other placements as well. cc @pmcpinto